test/XRP/issued-currency-client-integration.test.ts

```diff
@@ -573,133 +589,110 @@ describe('IssuedCurrencyClient Integration Tests', function (): void {
   > {
     this.timeout(timeoutMs)
-    const issuer = await XRPTestUtils.randomWalletFromFaucet()
-    const accountToUnfreeze = await XRPTestUtils.randomWalletFromFaucet()
-
     // GIVEN an existing issuer account who has a frozen trust line with a counter-party
-    await issuedCurrencyClient.requireAuthorizedTrustlines(issuer)
-
-    const trustLineCurrency = 'USD'
+    const trustLineCurrency = 'FRZ'
     await issuedCurrencyClient.authorizeTrustLine(
-      accountToUnfreeze.getAddress(),
+      walletMightHaveTrustLines.getAddress(),
       trustLineCurrency,
-      issuer,
+      issuerWalletAuthTrustLines,
     )
 
     await issuedCurrencyClient.freezeTrustLine(
-      accountToUnfreeze.getAddress(),
+      walletMightHaveTrustLines.getAddress(),
       trustLineCurrency,
-      issuer,
+      issuerWalletAuthTrustLines,
     )
 
     // WHEN the issuer unfreezes the trustline
     await issuedCurrencyClient.unfreezeTrustLine(
-      accountToUnfreeze.getAddress(),
+      walletMightHaveTrustLines.getAddress(),
       trustLineCurrency,
-      issuer,
+      issuerWalletAuthTrustLines,
     )
 
     const trustLines = await issuedCurrencyClient.getTrustLines(
-      issuer.getAddress(),
+      issuerWalletAuthTrustLines.getAddress(),
     )
 
-    const [unfrozenTrustLine] = trustLines
+    let unfrozenTrustLine
+    trustLines.forEach((trustLine) => {
+      if (trustLine.currency === trustLineCurrency) {
+        unfrozenTrustLine = trustLine
+      }
+    })
 
     // THEN the trust line is not frozen.
     assert.equal(unfrozenTrustLine.freeze, false)
     assert.equal(unfrozenTrustLine.limit, '0')
   })
 
-  it('disableRipplingForTrustLine - sets no rippling on trust line', async function (): Promise<
+  it('disableRipplingForTrustLine/enableRipplingForTrustLine', async function (): Promise<
     void
   > {
     this.timeout(timeoutMs)
-    const issuer = await XRPTestUtils.randomWalletFromFaucet()
-    const trustLinePeerAccount = await XRPTestUtils.randomWalletFromFaucet()
-
     // GIVEN an existing issuer account who has a trust line with a counter-party
-    await issuedCurrencyClient.requireAuthorizedTrustlines(issuer)
-
-    const trustLineCurrency = 'USD'
+    const trustLineCurrency = 'NRP'
     await issuedCurrencyClient.authorizeTrustLine(
-      trustLinePeerAccount.getAddress(),
+      walletMightHaveTrustLines.getAddress(),
       trustLineCurrency,
-      issuer,
+      issuerWalletAuthTrustLines,
     )
 
     const trustLineAmount = '1'
 
     // WHEN the issuer sets no rippling on the trust line
     await issuedCurrencyClient.disableRipplingForTrustLine(
-      trustLinePeerAccount.getAddress(),
+      walletMightHaveTrustLines.getAddress(),
       trustLineCurrency,
       trustLineAmount,
-      issuer,
+      issuerWalletAuthTrustLines,
     )
 
-    const trustLines = await issuedCurrencyClient.getTrustLines(
-      issuer.getAddress(),
+    let trustLines = await issuedCurrencyClient.getTrustLines(
```

Comment:
this can still be `const` I believe, right?

Reply:
in this case I actually re-assign to trustLines later because this is a combined test case.  the linter will actually force it to be const if it's unchanging.